internal/contracts/manager.go

```diff
@@ -592,3 +593,10 @@ func (cm *contractManager) checkParamSchema(ctx context.Context, input interface
 	}
 	return nil
 }
+
+func (cm *contractManager) GenerateFFI(ctx context.Context, ns string, generationRequest *fftypes.FFIGenerationRequest) (*fftypes.FFI, error) {
+	if generationRequest.Namespace == "" {
+		generationRequest.Namespace = ns
```

Comment:
I think in general we ignore any nested Namespace value in an input body (ie list it in "ignored" fields so that Swagger won't generate it), and always write/overwrite with the one from the path.

Reply:
Ah I didn't realize that. I did the exact opposite here. I'll switch it around to match the existing pattern. Thanks!